openbas-front/src/admin/components/common/injects/Injects.tsx

```diff
@@ -264,13 +267,24 @@ const Injects: FunctionComponent<Props> = ({
 
   const onUpdate = (result: { result: string; entities: { injects: Record<string, InjectStore> } }) => {
     if (result.entities) {
-      const updated = result.entities.injects[result.result];
-      setInjects(injects.map((i) => {
-        return (i.inject_id !== updated.inject_id ? i as InjectOutputType : (updated as InjectOutputType));
-      }));
+      const updatedResults = result.entities.injects[result.result];
+      if (Array.isArray(updatedResults)) {
+        // Case of bulk update
+        injects.map(originalInject => updatedResults.find(updatedInject => updatedInject.inject_id === originalInject.inject_id) || originalInject);
```

Comment:
Just curious about this.
you don't need to push the result in a var ?

Reply:
Absolutely right, i broke the single update by doing that. Fixed. Thanks for spotting it !